eng/Signing.props

```diff
@@ -87,4 +82,44 @@
     <ItemsToSign Remove="$(ArtifactsPackagesDir)**\*tests*.nupkg" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(EnableDefaultArtifacts)' != 'true'">
+    <Artifact Include="$(ArtifactsPackagesDir)**\VS.Tools.*.nupkg;
+                       $(ArtifactsPackagesDir)**\VS.Redist.*.nupkg"
+              IsShipping="$([System.String]::Copy('%(RecursiveDir)').StartsWith('Shipping'))"
+              PublishFlatContainer="false" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(DotNetPublishUsingPipelines)' == 'true' and ('$(DotNetBuildPass)' == '' or '$(DotNetBuildPass)' == '1')">
```

Comment:
This condition looks unexpected to me. Why do you need `DotNetPublishUsingPipelines`?

Reply:
I just copied the condition from the previous case. I can remove it.